### PR TITLE
SRV-213 import inventory with classification

### PR DESCRIPTION
**Pull request by c3zi**

### CODE REVIEW

File: app/Client/LocalPublisherBannerClassifier.php [hunk 1]
```diff
@@ -38,24 +38,25 @@ public function __construct(ClassifierInterface $classifier)
...
+
+        foreach ($bannersInternalIds as $bannerId) {
             try {
                 $classificationCollection = $this->classifier->fetch($bannerId);
 
                 /** @var Classification $classification */
                 foreach ($classificationCollection as $classification) {
                     $collection->addClassification(
-                        $bannerId,
+                        array_search($bannerId, $bannersInternalIds, true),
```
Comment (PawelPodkalicki) [inline app/Client/LocalPublisherBannerClassifier.php]: $bannerId is one of $bannersInternalIds, so what is the reason behind this call?

File: app/Client/LocalPublisherBannerClassifier.php [hunk 2]
```diff
@@ -38,24 +38,25 @@ public function __construct(ClassifierInterface $classifier)
...
                         $classification->keyword(),
                         $classification->signature()
                     );
                 }
             } catch (BannerNotVerifiedException $exception) {
-                $collection->addEmptyClassification($bannerId);
+                $collection->addEmptyClassification(array_search($bannerId, $bannersInternalIds, true));
```
Comment (PawelPodkalicki) [inline app/Client/LocalPublisherBannerClassifier.php]: Same as above: $bannerId is one of $bannersInternalIds, so what is the reason behind this call?

File: routes/manager.php [hunk 3]
```diff
@@ -88,5 +88,6 @@
 Route::get('sites/stats/table/{date_start}/{date_end}', [StatsController::class, 'publisherStats']);
 Route::get('sites/stats/table2/{date_start}/{date_end}', [StatsController::class, 'publisherStatsWithTotal']);
 
+Route::patch('classifications/{site_id?}', [ClassifierController::class, 'add']);
```
Comment (PawelPodkalicki) [inline routes/manager.php]: This endpoint is already defined two lines below. Please, delete this line.

**Review by PawelPodkalicki — CHANGES_REQUESTED** [review]

File: app/Client/LocalPublisherBannerClassifier.php (hunk 1 above)
Comment (c3zi) [reply app/Client/LocalPublisherBannerClassifier.php]: `array_search` was removed regarding the fact `publicId` exists as a key :)

File: routes/manager.php (hunk 3 above)
Comment (c3zi) [reply routes/manager.php]: exactly :)

**Review by PawelPodkalicki — APPROVED** [review]

**Pull request merged**